src/socks-to-rtc/socks-to-rtc.ts

```diff
@@ -437,8 +437,12 @@ module SocksToRtc {
       return this.tcpConnection_.receiveNext()
         .then(socks.interpretRequestBuffer)
         .then((request:socks.Request) => {
-          log.info('%1: received endpoint from SOCKS client: %2', [
-              this.longId(), JSON.stringify(request.endpoint)]);
+          // The domain name is very sensitive, so we keep it out of the
+          // info-level logs, which may be uploaded.
+          log.debug('%1: received address from SOCKS client: %2', [
+              this.longId(), request.endpoint.address]);
+          log.info('%1: received port from SOCKS client: %2', [
```

Comment:
This is probably going to be a lot of 80 and 443 right? What do you think about just dropping this info message, and print the full address at debug level.  Likewise with rtc to net?